v_5_2_0/files/k8s-resource/calico-policy-only.yaml

```diff
@@ -0,0 +1,515 @@
+---
```

Comment:
I guess we have something similar in azure-operator now. It would be really good to have that aligned.